src/axom/multimat/multimat.hpp

```diff
@@ -629,13 +783,82 @@ class MultiMat
   // dense layout bivariate sets
   axom::Array<ProductSetType> m_denseBivarSet;
 
+  struct FieldBacking
+  {
+  private:
+    bool m_isOwned {false};
+
+    axom::Array<unsigned char> m_ucharData;
+    axom::Array<int> m_intData;
```

Comment:
Should these fixed std types instead of `unsigned char` and `int`?
E.g.  `std::uint8_t` and `std::int32_t`

(The answer could be `no`).